eras/conway/impl/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.7.0.0
 
+* Add `ConwayGovCertEnv`
```

Comment:
I also added a `GovCertEnv` in my local work, but named it without the `Conway` prefix. I like it better with the prefix, though I thought it was inconsistent with the other `Env`s we have. Nice! :raised_hands: